egress/types.go

```diff
@@ -35,12 +36,20 @@ type EncodedOutput interface {
 	GetFileOutputs() []*livekit.EncodedFileOutput
 	GetStreamOutputs() []*livekit.StreamOutput
 	GetSegmentOutputs() []*livekit.SegmentedFileOutput
+	ImageOutput
 }
 
 type EncodedOutputDeprecated interface {
```

Comment:
This is for deprecated requests which don't implement `Get_Outputs()`

Reply:
I think it's ok to leave it out of the legacy API? i.e. if you want new features then use the new API.